content/docs/start/data-management/data-and-model-access.md

```diff
@@ -1,23 +1,32 @@
 ---
 title: 'Get Started: Data and Model Access'
+description: 'Get started with accessing data and models with DVC. Learn how to
+bring, explore, and access data artifacts from outside the project'
 ---
 
 # Get Started: Data and Model Access
 
+<details>
+
+## 🎬 Click to watch a video intro.
+
+https://youtu.be/EE7Gk84OZY8
+
+</details>
```

Comment:
Proposal: Hiding the videos (moved right under the title) saves a lot of screen space, e.g.:

<img width="1236" alt="image" src="https://user-images.githubusercontent.com/1477535/194437281-3b83792d-d38f-4ed8-b62e-83cfbf117d39.png">

> Another alternative would be to put the thumbnails in the right sidebar (moved larger into the middle of the screen when clicked), but this would require building the feature into the docs engine first.

Reply:
> Another alternative would be to put small video thumbnails in the right sidebar (moved larger into the middle of the screen when clicked), but this would require building the feature into the docs engine first.

Just repeating that comment in case it wasn't noticed ☝🏼